tests/backup/backup_test.go

```diff
@@ -6378,6 +6378,8 @@ var _ = Describe("{DeleteUsersRole}", func() {
 				dash.VerifyFatal(err, nil, fmt.Sprintf("Removing role [%s] from the user [%s]", role, userName))
 			}
 		})
+		//Waiting for catalog to update before moving for validation
```

Comment:
Leave a space after the `//`

Also, it is not catalog. It should be keycloak

Reply:
I mean to say px-backup catalog

Reply:
What do you mean by Px-Backup catalog?